tests/testthat/test_paramtest_aorsf_surv_aorsf.R

```diff
@@ -7,6 +7,10 @@ test_that("paramtest surv.aorsf train", {
     "weights", # handled internally
     "oobag_fun", # handled internally
     "data", # handled internally
+    "n_thread", # handled internally
```

Comment:
The results of the parameter tests for the 'train' parameters are: 

Missing parameters:
- 'n_thread'
- 'sample_with_replacement'
- 'sample_fraction'
- 'split_rule'

This means they are exposed through the `aorfs::orsf()` function, so I don't understand why you add them to the `exclude` argument of the parameter test?

Reply:
thanks!